code/datums/position_point_vector.dm

```diff
@@ -151,6 +151,14 @@
 	starting_y = y
 	starting_z = z
 
+/datum/point/vector/proc/set_location(tile_x, tile_y, tile_z, p_x = 0, p_y = 0) // initialize_location of /datum/point/, doesn't set starting_x/y/z
+	if(!isnull(tile_x))
+		x = ((tile_x - 1) * world.icon_size) + world.icon_size / 2 + p_x + 1
+	if(!isnull(tile_y))
+		y = ((tile_y - 1) * world.icon_size) + world.icon_size / 2 + p_y + 1
```

Comment:
From [contributing guidelines](https://github.com/tgstation/tgstation/blob/master/.github/CONTRIBUTING.md#other-notes): "Do not divide when you can easily convert it to multiplication."

```suggestion
		x = ((tile_x - 1) * world.icon_size) + world.icon_size * 0.5 + p_x + 1
	if(!isnull(tile_y))
		y = ((tile_y - 1) * world.icon_size) + world.icon_size * 0.5 + p_y + 1
```

Reply:
I see, I'll scour the file for other needless divisions